src/everest/config/sampler_config.py

```diff
@@ -26,7 +26,7 @@ class SamplerConfig(BaseModel):  # type: ignore
 """,
     )
     method: str = Field(
-        default="default",
+        default="dakota/default",
```

Comment:
I suggest not giving a default here, because samplers are not specified by default anyway in the main config. If the user does specify one, they better know what method they want to use and should specify it explicitly.